tests/test_default_qubit.py

```diff
@@ -383,6 +392,7 @@ def test_ev(self):
 
         # loop through all supported observables
         for name, fn in self.dev._expectation_map.items():
+            print(name)
```

Comment:
This loop is a major reason I want to eventually port this test to pytest :slightly_smiling_face:

Reply:
I will refactor the entire `test_default_qubit`and port to pytest